src/mount/direntry_cache.h

```diff
@@ -31,6 +31,10 @@
 #include <boost/intrusive/list.hpp>
 #include <boost/intrusive/set.hpp>
 
+#define INVALID_INDEX 0xffff
+#define INVALID_PARENT 0xffffffff
+#define EMPTY_NAME ""
+
```

Comment:
Consider something like this:
```cpp
#include <limits>

constexpr uint32_t kInvalidIndex = std::numeric_limits<uint32_t>::max();
constexpr uint64_t kInvalidParent = std::numeric_limits<uint64_t>::max();
constexpr char kEmptyName[] = "";
```
This is a more modern and type-safe approach in C++. Plus, they are still compile-time constants like the #define macros.

Reply:
Applied.